src/components/seekbar/seekbar.tsx

```diff
@@ -98,6 +98,11 @@ class SeekBar extends Component<any, any> {
     }
   ];
 
+  updateSeekBarClientRect = () => {
```

Comment:
name of method similar with method from props, I would recommend rename to `handleUpdateSeekBarClientRect`, etc to simplify code reading